packages/block-editor/src/components/global-styles/shadow-panel-components.js

```diff
@@ -37,6 +30,11 @@ export function ShadowPopoverContainer( { shadow, onShadowChange, settings } ) {
 					activeShadow={ shadow }
 					onSelect={ onShadowChange }
 				/>
+				<div className="block-editor-global-styles__clear-shadow">
+					<ClearShadowButton
```

Comment:
Since this component will only be called once here, we might want to hardcode the component here.

Reply:
moved the component code inline.